utils/dist_build.sh

```diff
@@ -117,39 +122,44 @@ tar -pczf $amd64TGZDIR/../tyk-linux-amd64-$VERSION.tar.gz tyk.linux.amd64-$VERSI
 cd $armTGZDIR/../
 tar -pczf $armTGZDIR/../tyk-linux-arm64-$VERSION.tar.gz tyk.linux.arm64-$VERSION/
 
-echo "Removing old builds"
-rm -f *.deb
-rm -f *.rpm
+if [ $BUILDPKGS == "1" ]; then
+    CONFIGFILES=(--config-files /opt/tyk-gateway/apps --config-files /opt/tyk-gateway/templates --config-files /opt/tyk-gateway/middleware --config-files /opt/tyk-gateway/event_handlers --config-files /opt/tyk-gateway/js --config-files /opt/tyk-gateway/policies --config-files /opt/tyk-gateway/tyk.conf)
+    FPMCOMMON=(--name tyk-gateway --description "$DESCRIPTION" -v $VERSION --vendor "Tyk Technologies Ltd" -m "<info@tyk.io>" --url "https://tyk.io" -s dir)
+
+    echo "Removing old builds"
+    rm -f *.deb
+    rm -f *.rpm
 
-echo Creating Deb Package for AMD64
-cd $amd64TGZDIR/
-fpm -n tyk-gateway --description "$DESCRIPTION" -v $VERSION  --after-install $amd64TGZDIR/install/post_install.sh --after-remove $amd64TGZDIR/install/post_remove.sh -a amd64 -s dir -t deb ./=/opt/tyk-gateway
-fpm -n tyk-gateway --description "$DESCRIPTION" -v $VERSION  --after-install $amd64TGZDIR/install/post_install.sh --after-remove $amd64TGZDIR/install/post_remove.sh -a amd64 -s dir -t rpm ./=/opt/tyk-gateway
+    echo Creating Deb Package for AMD64
+    cd $amd64TGZDIR/
+    fpm "${FPMCOMMON[@]}" --after-install $amd64TGZDIR/install/post_install.sh --after-remove $amd64TGZDIR/install/post_remove.sh -a amd64 -t deb "${CONFIGFILES[@]}" ./=/opt/tyk-gateway
+    fpm "${FPMCOMMON[@]}" -x **/*.deb --after-install $amd64TGZDIR/install/post_install.sh --after-remove $amd64TGZDIR/install/post_remove.sh -a amd64 -t rpm "${CONFIGFILES[@]}" ./=/opt/tyk-gateway
```

Comment:
FYI: Previously RPM packages also included the DEB package inside it so now it's about half in size :)